blog/2023/06/PSA-dont-use-threadid.md

```diff
@@ -0,0 +1,175 @@
++++
+mintoclevel = 2
+maxtoclevel = 3
+title = "PSA: Thread-local state is no longer recommended; Common misconceptions about threadid() and nthreads()"
+authors = "Mason Protter, Valentin Churavy, Ian Butterworth, ..."
+published = "30 June 2023"
+rss_pubdate = Date(2023, 06, 30)
+rss = """PSA: Thread-local state is no longer recommended; Common misconceptions about threadid() and nthreads()"""
++++
+
+\toc
+
```

Comment:
the `toc` should probably have a listing for the first section. Maybe we can just do
```suggestion
\toc

# The buggy pattern
```
or something>